rclcpp_lifecycle/include/rclcpp_lifecycle/lifecycle_node.hpp

```diff
@@ -1087,6 +1088,8 @@ class LifecycleNode : public node_interfaces::LifecycleNodeInterface,
   rclcpp::node_interfaces::NodeTimeSourceInterface::SharedPtr node_time_source_;
   rclcpp::node_interfaces::NodeWaitablesInterface::SharedPtr node_waitables_;
 
+  rclcpp::NodeBuiltinExecutor::SharedPtr node_builtin_executor_;
```

Comment:
It should be updated as well.

Reply:
Yes. Updated before pushing to github rclcpp